templates/tags/SageFinancialProgramCodesOther.json

```diff
@@ -51,6 +51,7 @@
   "PECDCC-UMass / 106000",
   "PKU App-BioMarin / 613400",
   "Psorcast Pfizer / 613600",
+  "RECOVER / 122500",
```

Comment:
We already have `MGH RECOVER / 122500` above on line 36, which matches the name in our financial system

Reply:
welp, good thing we review stuff here. thanks @jesusaurus!  

@itismeghasyam i'm guessing that you can use the existing one and I'll just close out this PR.

Reply:
I will change it to 'MGH RECOVER /  122500'. Thanks!